tools/src/main/java/org/apache/rocketmq/tools/command/broker/UpdateBrokerConfigSubCommand.java

```diff
@@ -117,4 +119,23 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook) t
             defaultMQAdminExt.shutdown();
         }
     }
+
+    public Properties getUpdatedConfig(CommandLine commandLine) {
```

Comment:
Is it possible to put this function implementation into CommandUtils？ so that it will not be written twice.